api/src/main/java/com.epam.demo/App.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.demo;
+
+
+public class App {
+    public static void main(String[] args) {
```

Comment:
make sure you can run your project from command line with different arguments provided: if no arguments are provided then default numbers for run should be "12", "79". Please notify user with output like "Running wihout arguments, default values "12", "79" will be used", "Run with arguments "15", "-28". Please handle a case when not a number is in input line.

Reply:
also create an integration test with several cases on input provided.